cornichon-docs/src/main/tut/dsl.md

```diff
@@ -274,6 +274,85 @@ And assert body.ignoring("city", "publisher").is(
 ```
 
 
+## Kafka Support (experimental)
+
```

Comment:
Could you add a line saying `this module is experimental and will be subject to important changes in the future`

Reply:
added disclaimer to Doc